src/datePicker/DatePicker.tsx

```diff
@@ -180,6 +181,7 @@ export class DatePicker extends PureComponent<DatePickerProps, DatePickerState>
           months={months}
           firstDayOfWeek={firstDayOfWeek}
           localeUtils={{ ...DayPicker.LocaleUtils, formatMonthTitle: this.formatMonthTitle }}
+          initialMonth={initialDate}
```

Comment:
unit test?

Reply:
The feature is from the `react-day-picker` lib so it's not our job to test it. Unless you want me to test that we pass the prop each time?

Reply:
Looks like we didn't do that for DatePicker but we normally do check that our props are forwared in other components. I'll approve it and leave it up to you.